docs/content/02-tutorials/08-custom-domain.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "h. Setup Custom Domain 🔗"
+weight = 28
++++
+
+Custom domains are a great way to shorten the default API Gateway domain into something more readable. For example instead of `https://sdrojbaf64.execute-api.us-east-2.amazonaws.com/` users could navigate to `example.com`.
+
+Before getting started ensure that you:
+
+* Own a Domain
+* Know how to change basic DNS settings
+
+## Create new Domain in API Gateway
+
+1. Navigate to [API Gateway Console > Custom domain names > Create](https://console.aws.amazon.com/apigateway/main/publish/domain-names/create?)
+
+    ![Create Domain](08-custom-domain/create-domain.png)
+
+2. If you haven't already created a certificate, you can do that through ACM by clicking "Create new certificate in Amazon Certificate Manager". If you already have a cert, skip to [Setup API Mappings](#setup-api-mappings).
+
+## Create certificate
+
+1. On the [Certificate Manager console](https://console.aws.amazon.com/acm/home), click "Request"
+2. Enter in the domain you own, i.e. `example.com`
+3. Select either email or DNS validation (I prefer email)
+4. Click request
+
+    ![Request Certificate](08-custom-domain/certificate.png)
+
+If you selected email, you'll get an email to the email you have on file with the regisitrar. Click "I approve" to activate the certificate.
+
+## Setup API Mappings
+
+Return back to the API Gateway console, select the certificate and click "Create" to complete the setup.
+
+1. Next on the API Gateway > Custom domain names > your domain
+2. Click on **Configure API Mappings** and create a new one like:
+
+    ![API Gateway Custom Mappings](08-custom-domain/api-mappings.png)
+
+You'll see a section called **API Gateway domain name**. Copy that for DNS setup.
+
+## Setup DNS
+
+Create a DNS **CNAME** rule that points your domain to the API Gateway domain like so:
+
+| Rule  | Source      | Destination                                       |
+|-------|-------------|---------------------------------------------------|
+| CNAME | example.com | d-hb4qd1edp8.execute-api.us-east-2.amazonaws.com. |
+
+**Note:** this is not the same as the URL you access when you go to PCM. Check API Gateway > Custom Domains > API Gateway domain name for the correct domain.
+
+## Add the domain to your Cognito user pool
+
+1. Navigate to the [Cognito Console](https://console.aws.amazon.com/cognito/v2/idp/user-pools) > Select your user pool
+2. Under App Integration > Domain > **Create custom domain**
+
+    ![Cognito Custom Domain](08-custom-domain/cognito-custom-domain.png)
+
+3. Enter in the same domain as before and select the certificate
+
+## Disable Custom Domains in API Gateway
```

Comment:
Are you going to work on this chapter as well?

Reply:
done ✅